src/main/java/org/openmrs/isanteplus/performancedata/options/GeneratorOptions.java

```diff
@@ -91,6 +95,10 @@ public long getVisits() {
         return visitOption.getVisits();
     }
 
+    public long getEncounters() {
```

Comment:
shouldn't it be getEncountersNumber? or size?

Reply:
toString of UUID does not match mysql insert syntax. I can wrap it up in some custom class and override toString method, but String works just fine as well.

Reply:
ok

Reply:
up?